tag/nitag.yml

```diff
@@ -1525,6 +1483,66 @@ definitions:
         max: "5.0"
         avg: 2.0
         count: 5
+  TagPathWithCurrentValue:
+    description: Tag path with timestamped value
+    type: object
+    title: Tag Path With Current Value
+    properties:
+      path:
+        type: string
+        description: Tag path
+      current:
+        $ref: '#/definitions/TimestampedTagValue'
+  TagPathWithValue:
+    description: Tag path with value
+    type: object
+    title: Tag Path With Value
+    properties:
+      path:
+        type: string
+        description: Tag path
+      current:
+        $ref: '#/definitions/TagValue'
+  TagPathAndTimestamp:
+    type: object
+    title: Tag Path and Timestamp
+    description: A tag path and the timestamp of its current value
+    properties:
+      timestamp:
+        description: Timestamp associated with the current value
+        type: string
+        format: iso-date-time
+        example: '2018-09-04T18:45:08Z'
+      path:
+        type: string
+  TagPathWithMaxAggregate:
+    type: object
+    title: Tag Path With Max Aggregate
+    properties:
+      path:
+        type: string
+      max:
+        description: Maximum value from the tag's aggregates
+        type: string
+  TagPathWithMinAggregate:
+    type: object
+    title: Tag Path with Min Aggregate
+    properties:
+      path:
+        type: string
+      min:
+        description: Minimum value from the tag's aggregates
+        type: string
+  TagPathAndMeanValue:
```

Comment:
Could we name this "TagPathWithMeanAggregate" instead? That would be more consistent with the other new types. I realize that the titles were already inconsistent, and that's what gave rise to the type names. Feel free to drop if there's pressure to just get the codegen fixed quickly.